distribution/kernel/pom.xml

```diff
@@ -249,10 +249,10 @@
                 </executions>
             </plugin>
             <!--
-      Uses the assembly descriptor in this DDF distributon project (bin.xml) which
-      also injects the copied common-bin.xml descriptor from ddf-common, to create
-      the ddf-core-xxx.zip distribution in the /target directory.
-      -->
+            Uses the assembly descriptor in this DDF distributon project (bin.xml) which
+            also injects the copied common-bin.xml descriptor from ddf-common, to create
+            the ddf-core-xxx.zip distribution in the /target directory.
```

Comment:
✏️ Comment should probably say _ddf-kernel-xxx.zip_ instead of _ddf-core-xxx.zip_.